src/Okta.Sdk/Internal/UserAgentBuilder.cs

```diff
@@ -39,11 +43,36 @@ public UserAgentBuilder()
 
         private string Generate()
         {
-            var sdkToken = $"{OktaSdkUserAgentName}/{GetSdkVersion()}";
+            string sdkToken = string.Empty;
+            string runtimeToken = string.Empty;
+            string operatingSystemToken = string.Empty;
 
-            var runtimeToken = $"runtime/{Sanitize(RuntimeInformation.FrameworkDescription)}";
+            try
+            {
+                sdkToken = $"{OktaSdkUserAgentName}/{GetSdkVersion()}";
+            }
+            catch (Exception ex)
+            {
+                _logger.LogDebug($"An error occurred generating the {nameof(sdkToken)} portion of the user-agent string.  Exception: {ex.Message}");
```

Comment:
I think you should probably go with `LogError` instead of `LogDebug` unless you are really expecting this to fail often in common scenarios. Messages logged with `LogDebug` are considered very verbose and are often not even displayed or recorded unless the logging level is changed from its default.

Reply:
> Messages logged with LogDebug are considered very verbose and are often not even displayed or recorded unless the logging level is changed from its default.

That's why I chose debug.  I don't think it makes sense for people to see this error unless they're digging deep.  The 99% just want their API call to work.  Seeing this logged as an error seems confusing when the function performed its purpose (i.e., to the developer, there wasn't an error).  My two cents.

Reply:
Hmmm, OK that makes sense to me. Thanks!